framework/tests/viewpointtest/LostNonReflexive.java

```diff
@@ -0,0 +1,37 @@
+import viewpointtest.quals.*;
+
+public class LostNonReflexive {
+    @ReceiverDependentQual Object f;
+
+    @SuppressWarnings({"inconsistent.constructor.type", "super.invocation.invalid"})
+    @ReceiverDependentQual
+    LostNonReflexive(@ReceiverDependentQual Object args) {}
+
+    @ReceiverDependentQual
```

Comment:
In a separate PR, can you ensure these test qualifiers are added to type annotation formatting: https://github.com/eisop/checker-framework/blob/930212447c1a39bc9cdfa82b4b0b690a9739b27b/build.gradle#L303C36-L303C73

Maybe there is some external way to set this in Spotless? Or some other way to do this more nicely.

Reply:
I am not sure, let me check...

Reply:
I will leave this to new issue #873